masterlist.yaml

```
@@ -808,3 +808,28 @@ plugins:
       - 'NOM3.0.esp'
       - 'NOM3_PT SO.esp'
       - 'NoM_Creatures Loot Standard.esp'
+
+  ### Other Stuff (sort later) ###
+  - name: 'Better Propylon Teleport Warp.ESP'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/46364/'
+        name: 'Pikas Miscellaneous Mods'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Better Propylon Teleport Warp ESP' ]
+        condition: 'many("Better Propylon Teleport Warp(-Master Index)?\.esp")'
+  - name: 'Better Propylon Teleport Warp-Master Index.esp'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/46364/'
+        name: 'Pikas Miscellaneous Mods'
+    after:
+      - 'master_index.esp'
+      - 'MasterIndexRedux.esp'
+    req:
+      - name: 'master_index.esp'
+        condition: 'not active("MasterIndexRedux.esp")'
+        display: '[Master Index Official Plugin](https://www.nexusmods.com/morrowind/mods/42985) or master-index.esp from [Unofficial Morrowind Official Plugins Patched.ESP](https://www.nexusmods.com/morrowind/mods/43931)'
```

The order of the `condition` & `display` keys should be reversed, & I've adjusted the display a bit to be more in line with similar displays in the masterlists:

```suggestion
        display: '[Master Index Plugin](https://www.nexusmods.com/morrowind/mods/42985/) **OR** [Unofficial Morrowind Official Plugins Patched](https://www.nexusmods.com/morrowind/mods/43931/) (**master-index.esp**)'
        condition: 'not active("MasterIndexRedux.esp")'
```